pkg/generators/models/testdata/cases/oneof_mapped_discriminator/expected/model_error.go

```diff
@@ -0,0 +1,137 @@
+// This file is auto-generated, DO NOT EDIT.
+//
+// Source:
+//     Title: Test oneOf discriminator support
+//     Version: 0.1.0
+package generatortest
+
+import (
+	"encoding/json"
+	"fmt"
+	validation "github.com/go-ozzo/ozzo-validation/v4"
+	"github.com/mitchellh/mapstructure"
+)
+
+type errorer interface {
+	GetKind() string
+	Validate() error
+}
+
+// Error is a oneOf type.
+type Error struct {
+	data errorer
```

Comment:
I created a private interface just for the oneOf and use this instead of a plan `interface{}`. I think this makes the internal code a little bit nicer